### DISCUSSION TITLE
how to unblock any website

### DISCUSSION THREAD

**Discussion by LieutenantGst** [S-PScripts/chromebook-utilities#13] [General]
How to unblock any site in a few minutes

1. Make a Google doc paper (docs.new)
2. Add some links like Spotify or Youtube
3. click any of the links 
6. Its going to work trust me works with securely!

please don't exit the website you used because if you try again it'll block it.

**Comment by S-PScripts** [comment]
i already have this, i also saw you posted it in 3kh0's discussions (his repo doesn't have it but mine does at https://github.com/S-PScripts/chromebook-utilities/blob/main/Exploits/Document%20Exploit
